src/test/resources/application-kafka.properties

```diff
@@ -1 +1,13 @@
-kafka.bootstrap-servers: ${spring.embedded.kafka.brokers}
\ No newline at end of file
+kafka.bootstrap-servers=${spring.embedded.kafka.brokers}
+cat-broker.route.cat-broker.from=kafka:{{consumer.topic.cat-broker}}?brokers={{kafka.bootstrap-servers}}
+cat-broker.route.cat-broker.cats-r-us=kafka:{{producer.topic.cats-r-us}}?brokers={{kafka.bootstrap-servers}}
+cat-broker.route.cat-broker.dlq=kafka:{{producer.topic.dlq}}?brokers={{kafka.bootstrap-servers}}
+
+kafka.serializerClass=kafka.serializer.StringEncoder
+
+# Producer properties
+producer.topic.cats-r-us=cats-r-us
+producer.topic.dlq=dlq
+
+# Consumer properties
+consumer.topic.cat-broker=cat-broker
```

Comment:
I'm suprised that you needed to dulicate the rest of src/main/resources/application-kafka.properties. The test only needs to set the url of the kafka broker, all the other properties are exaclty the same.

I thought spring could take src/main/resources/application-kafka.properties first, then override with the test version as its "more specific" src/main/resources/application-kafka.properties 

https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-profile-specific-properties

When I tested this locally without the duplicated lines the test fails though.